src/main/java/jarvis/utils/TextUI.java

```diff
@@ -0,0 +1,74 @@
+package jarvis.utils;
+
+import java.util.Scanner;
+
+public class TextUI {
+    private static final String LOGO = "\n"
+        + "    /$$$$$  /$$$$$$  /$$$$$$$  /$$    /$$ /$$$$$$  /$$$$$$\n"
+        + "   |__  $$ /$$__  $$| $$__  $$| $$   | $$|_  $$_/ /$$__  $$\n"
+        + "      | $$| $$  \\ $$| $$  \\ $$| $$   | $$  | $$  | $$  \\__/\n"
+        + "      | $$| $$$$$$$$| $$$$$$$/|  $$ / $$/  | $$  |  $$$$$$\n"
+        + " /$$  | $$| $$__  $$| $$__  $$ \\  $$ $$/   | $$   \\____  $$\n"
+        + "| $$  | $$| $$  | $$| $$  \\ $$  \\  $$$/    | $$   /$$  \\ $$\n"
+        + "|  $$$$$$/| $$  | $$| $$  | $$   \\  $/    /$$$$$$|  $$$$$$/\n"
+        + " \\______/ |__/  |__/|__/  |__/    \\_/    |______/ \\______/\n";
+    private static final String SEPARATOR = "____________________________________________________________";
+    private static final String EXIT_MSG = "Bye. Hope to see you again soon!";
+    private static final String GREET_MSG = "Hello! I'm Jarvis\nWhat can I do for you?";
+
+    private final Scanner sc;
+
+    public TextUI() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Get the next command.
+     *
+     * @return the next command
+     */
+    public String nextCmd() {
```

Comment:
Perhaps it would improve the code quality if the method name was used to describe what the method does. For example, changing the name to `getNextCmd` 🤔